internal/command/scale/count_machines.go

```diff
@@ -61,8 +62,12 @@ func runMachinesScaleCount(ctx context.Context, appName string, appConfig *appco
 		return err
 	}
 
+	defaultGuest := new(api.MachineGuest)
+	defaultGuest.SetSize(deploy.DefaultVMSize)
 	defaults := newDefaults(appConfig, latestCompleteRelease, machines, volumes,
-		flag.GetString(ctx, "from-snapshot"), flag.GetBool(ctx, "with-new-volumes"))
+		flag.GetString(ctx, "from-snapshot"), flag.GetBool(ctx, "with-new-volumes"), defaultGuest)
+
+	deploy.ApplyFlagsToGuest(ctx, defaults.guest)
```

Comment:
no need to access the private field of defaults when you can apply it to `defaultGuest` right after creating it.

Reply:
ah that's my mistake, i should've seen that